docs/index.rst

```diff
@@ -182,7 +182,6 @@ Project details
    :maxdepth: 1
 
    whatsnew/index
-   lts_policy
```

Comment:
Do we want to keep this page but on that page link to APE 21 and tell people that LTS is no more?

Reply:
Yes we could do this, will update